lesson13/index.js

```diff
@@ -0,0 +1,132 @@
+const contents = document.getElementById('js-contents');
+
+window.addEventListener('DOMContentLoaded', () => {
+    // モーダルを表示させるボタンを表示する
+    showModalOpenButton();
+});
+
+const showModalOpenButton = () => {
+    const openButton = document.createElement('button');
+    openButton.id = 'modalOpenButton';
+    openButton.textContent = 'クリックしたらモーダルが表示されます';
+    contents.appendChild(openButton);
+    openButton.addEventListener('click', showModal)
+}
+
+const removeModalOpenButton = () => {
+    const modalOpenButton = document.getElementById('modalOpenButton')
+    modalOpenButton.remove();
+}
+
+const showModal = () => {
+    // モーダルを表示するボタンを取り除く
+    removeModalOpenButton();
+
+    // モーダルを作成
+    const modal = document.createElement('div');
+    modal.id = 'modal';
+    contents.appendChild(modal);
+    const modalContent = document.createElement('div');
+    modalContent.id = 'modal-content';
+    modal.appendChild(modalContent);
+
+    // リクエスト送信ボタンを作成し、モーダルの中に入れる
+    const requestButton = document.createElement('button');
+    requestButton.id = 'requestButton';
+    requestButton.textContent = 'クリックしたらリクエストが送信されます';
+    modalContent.appendChild(requestButton);
+
+    // 閉じるボタンを作成し、モーダルの中に入れる
+    const closeModalButton = document.createElement('button');
+    closeModalButton.id = 'closeModal';
+    closeModalButton.textContent = '閉じる';
+    modalContent.appendChild(closeModalButton);
```

Comment:
[IMO]
Overall, the code is easy to read and I think it's very good!
Since the function of shouModal is a little long, I think that it will be neat if you make the requestButton and closeModalButton into a function and put a modal.

Reply:
Thank you for your reviewing!
I separated the function. -> 951dc3c

I returned the element created by the createRequestButton function and put it in the requestButton variable in the showModal function and used it. Is that okay?